src/compiler/moduleNameResolver.ts

```diff
@@ -298,6 +298,7 @@ namespace ts {
      * is assumed to be the same as root directory of the project.
      */
     export function resolveTypeReferenceDirective(typeReferenceDirectiveName: string, containingFile: string | undefined, options: CompilerOptions, host: ModuleResolutionHost, redirectedReference?: ResolvedProjectReference, cache?: TypeReferenceDirectiveResolutionCache, resolutionMode?: SourceFile["impliedNodeFormat"]): ResolvedTypeReferenceDirectiveWithFailedLookupLocations {
+        Debug.assert(typeof typeReferenceDirectiveName === "string", "Non-string value passed to `ts.resolveTypeReferenceDirective`, likely by an out-of-date wrapping package working with an outdated `resolveTypeReferenceDirectives` signature. This is probably not an problem in TS itself.");

```

Comment:
Recommend one typo fix and removing one almost-redundant adjective.

```suggestion
        Debug.assert(typeof typeReferenceDirectiveName === "string", "Non-string value passed to `ts.resolveTypeReferenceDirective`, likely by an wrapping package working with an outdated `resolveTypeReferenceDirectives` signature. This is probably not a problem in TS itself.");
```

Reply:
Fixed one typo and introduced another - "an wrapping package" 😛